agent/csi/mod_csi_docker.go

```diff
@@ -10,16 +11,47 @@ import (
 	"sync"
 
 	gofig "github.com/akutz/gofig/types"
+	"github.com/akutz/gotil"
 	"github.com/codedellemc/gocsi"
 	"github.com/codedellemc/gocsi/csi"
 	"github.com/codedellemc/gocsi/mount"
-	dvol "github.com/docker/go-plugins-helpers/volume"
 
 	"github.com/codedellemc/rexray/agent"
 	"github.com/codedellemc/rexray/core"
 	apictx "github.com/codedellemc/rexray/libstorage/api/context"
 	"github.com/codedellemc/rexray/libstorage/api/registry"
 	apitypes "github.com/codedellemc/rexray/libstorage/api/types"
+	dvol "github.com/docker/go-plugins-helpers/volume"
+)
+
+const (
+	rrCSIMountPath  = "rexray.csi.mount.path"
+	rrCSINFSVolumes = "rexray.csi.nfs.volumes"
+)
+
+func init() {
```

Comment:
This is duplicated with another `init` below. I can't believe Hound or any linter doesn't flag this. I hit the same thing in my rebase.